bench/blast/math/simd/Ger.cpp

```diff
@@ -12,17 +12,16 @@
 
 namespace blast :: benchmark
 {
-    template <typename T, size_t M, size_t N, bool SO>
+    template <typename T, size_t M, size_t N, StorageOrder SO>
     static void BM_RegisterMatrix_ger_nt(State& state)
     {
         using Kernel = RegisterMatrix<T, M, N, SO>;
```

Comment:
I only see this now, but technically a 'kernel' is the operation, i.e. `dgemm`, not the data you're operating on.

Reply:
True. This wording remains for histerical reasons.